profiles/common/modules/custom/nexteuropa_varnish/nexteuropa_varnish.module

```diff
@@ -83,8 +83,9 @@ function nexteuropa_varnish_cache_purge_rule_label($purge_rule) {
  */
 function nexteuropa_varnish_workbench_moderation_transition($node, $previous_state, $new_state) {
   $purge
-    = ($new_state == workbench_moderation_state_published()) ||
-      ($previous_state == workbench_moderation_state_published() && $new_state !== $previous_state);
+    = ($node->workbench_moderation['my_revision']->published) ||
```

Comment:
The PURGE request needs to be send in the following cases:
- change a moderation state to 'Publish' - ex. creation of a new page
- change a moderation state from 'Publish' to any other state - ex. unpublishing a page